UnitsNet.Tests/CustomCode/MassTests.cs

```diff
@@ -42,11 +42,11 @@ public class MassTests : MassTestsBase
 
         protected override double KilotonnesInOneKilogram => 1E-6;
 
-        protected override double LongTonsInOneKilogram => 0.000984207;
+        protected override double LongTonsInOneKilogram => 9.842065276110606e-4;
```

Comment:
This PR is getting a bit crowded and not so easy to review. Could you perhaps pull out the bits not directly related to absolute/relative functionality in a separate PR?

Reply:
Unfortunately all of these changes are so the tests pass. These validation numbers were off quite a bit when I changed the tests to validate by % tolerance (not precise enough).

Reply:
Haha 😆 it's so good to finally fix this!
I understand about the precision changes, no problem!

Reply:
Is this getting ready now? I might find some time tomorrow to review.

Reply:
Yup! I'm done making changes.